projects/packages/my-jetpack/_inc/components/stats-section/count-comparison-card.jsx

```diff
@@ -0,0 +1,95 @@
+import { numberFormat } from '@automattic/jetpack-components';
+import { Card } from '@wordpress/components';
+import { arrowDown, arrowUp, Icon } from '@wordpress/icons';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './style.module.scss';
+
+const subtract = ( a, b ) => {
+	return a === null || b === null || b === undefined ? null : a - b;
```

Comment:
What is `a = undefined`? I could think of the following suggestion to make it clearer. it will also not allow for any `null` or `undefined`, but also `string` etc
```suggestion
	if ( typeof a !== 'number' || typeof b !== 'number' ) {
	    return null;
	}
	
	return a - b;
```

Reply:
Yes, I just copied that over and didn't change.